src/main/webapp/app/exercises/text/participate/text-editor.component.ts

```diff
@@ -239,8 +239,8 @@ export class TextEditorComponent implements OnInit, OnDestroy, ComponentCanDeact
                     this.jhiAlertService.warning('entity.action.submitDeadlineMissedAlert');
                 }
             },
-            () => {
-                this.jhiAlertService.error('artemisApp.modelingEditor.error');
+            (err) => {
+                this.jhiAlertService.error(err.error.message);
```

Comment:
Can we make sure the error message is actually set (to what we expect)? How does this look/work when an error different to the one specific BadRequest happens?

Reply:
All errors we throw in this resource seem to have an associated message(with translations). Here's an example of something generic going wrong: 
![image](https://user-images.githubusercontent.com/44805696/125763602-119736bb-24e9-42a4-a119-2eeffb19f126.png)
If you want I can add a check nonetheless.

Reply:
If all other errors have messages with translations this is fine. Thanks for clarifying! 👍